docs/source/index.rst

```diff
@@ -20,12 +20,6 @@ Lightly has been built to help you understand and work with large unlabeled
 datasets. It is built on top of PyTorch and therefore fully compatible with 
 other frameworks such as Fast.ai.
 
-**NEW** Lightly now has integrated support for active learning in combination 
```

Comment:
Only do the changes in this file. Do not do any changes in other files

Reply:
this is not the only file changed